todoman/model.py

```diff
@@ -585,8 +586,11 @@ def todos(self, all=False, lists=[], urgent=False, location='',
             q = ', '.join(['?'] * len(lists))
             extra_where.append('AND files.list_name IN ({})'.format(q))
             params.extend(lists)
-        if urgent:
-            extra_where.append('AND priority = 9')
+        if priority:
+            if priority == 'medium' or priority == '!low':
+                extra_where.append('AND priority = 5')
+            elif priority == 'high':
+                extra_where.append('AND priority = 1')
```

Comment:
I would always just choose the highest possible number, i.e. `9` for low, `4` for high. This makes this particular code easier to write.